ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -408,50 +422,119 @@ device_classes: !ZenPackSpec
               Verify Resize:
                 dpName: openstack_serverVerifyResizeCount
                 format: '%7.0lf'
+          Volumes:
+            units: count
+            miny: 0
+            graphpoints:
+              Active:
+                dpName: openstack_volumeActiveCount
+                format: '%7.0lf'
+              Bootable:
+                dpName: openstack_volumeBootableCount
+                format: '%7.0lf'
+              Attached:
+                dpName: openstack_volumeAttachedCount
+                format: '%7.0lf'
+              Available:
+                dpName: openstack_volumeAvailableCount
+                format: '%7.0lf'
+              InUse:
+                dpName: openstack_volumeInUseCount
+                format: '%7.0lf'
+              Total:
+                dpName: openstack_volumeTotalCount
+                format: '%7.0lf'
+              Unknown:
+                dpName: openstack_volumeUnknownCount
+                format: '%7.0lf'
+          Snapshots:
+            units: count
+            miny: 0
+            graphpoints:
+              Total:
+                dpName: openstack_snapshotTotalCount
+                format: '%7.0lf'
+              Available:
+                dpName: openstack_snapshotAvailableCount
+                format: '%7.0lf'
+              In Progress:
+                dpName: openstack_snapshotInProgressCount
+                format: '%7.0lf'
+          Pools:
+            units: count
+            miny: 0
+            graphpoints:
+              Total:
+                dpName: openstack_poolTotalCount
+                format: '%7.0lf'
+              Thin Provision:
+                dpName: openstack_poolThinProvisioningSupportCount
+                format: '%7.0lf'
+              Thick Provision:
+                dpName: openstack_poolThickProvisioningSupportCount
+                format: '%7.0lf'
+              QoS:
+                dpName: openstack_poolQoSSupportCount
+                format: '%7.0lf'
       Instance:
         description: 'OpenStackInfrastructure: Instance Monitoring'
         targetPythonClass: ZenPacks.zenoss.OpenStackInfrastructure.Instance
         datasources:
           cpuUtilization:
-            type: OpenStack Ceilometer Perf AMQP
+#            type: OpenStack Ceilometer Perf AMQP
+            type: OpenStack Ceilometer Perf API
```

Comment:
Huge red flag here.

We can not switch from AMQP to the API at this time.  Please revert these changes.